packages/core/stories/toolkit-provider.doc.stories.mdx

```diff
@@ -220,3 +220,32 @@ Then we would see the following HTML.
   </body>
 </html>
 ```
+
+By default, the root level `ToolkitProvider` will apply the theme and density classes and the theme mode data attribute to the `<html>` element and any nested `ToolkitProvider` will create a div with the `uitk-provider` class
+which the classes and data attributes are added,
+if you want the root level `ToolkitProvider` to create a div instead, then pass the value "scope" in the `applyClassesTo`
```

Comment:
not a major issue, but to me this is a bit wordy, could split into 2 sentences so it's easier to consume

Reply:
`which the classes and data attributes are added` probably is a typo